### PR TITLE
Basic general docker setup for workshop

### PR DESCRIPTION
**Pull request by PascalLike**
Got the composition code with elastic from [here](https://github.com/geopython/pygeoapi/tree/master/docker/examples/elastic).

Added some extra example data (for records)

Added a README with operating instructions for docker-composition

### CODE REVIEW

File: workshop/docker/data/README.md [hunk 1]
```diff
@@ -8,12 +8,22 @@ This directory provides sample data to demonstrate functionality.
...
+
+### Free Wifi Hotspots in Florence
+- title: [free-wifi-florence.xml](./metadata/xml/free-wifi-florence.xml)
+- url: https://opendata.comune.fi.it/?q=metarepo/datasetinfo&id=fb5b7bac-bcb0-4326-9388-7e3f3d671d71
+- description: The dataset shows the location of the places in the Municipality of Florence where a free wireless internet connection service (Wifi) is available.
+- copyright: Comune di Firenze
+- license: CC-BY 4.0
+
+### Carsharing spots in Florence
+- title: [carsharing-florence.xml](./metadata/xml/carsharing-florence.xml)
```
Comment (tomkralidis) [inline workshop/docker/data/README.md:25]: Should this also be ` - title: [carsharing-florence.xml](./records/xml/carsharing-florence.xml)` ?

File: workshop/docker/data/records/example_catalog.geojson [hunk 2]
```diff
@@ -0,0 +1,252 @@
+{
```
Comment (tomkralidis) [inline workshop/docker/data/records/example_catalog.geojson:1]: Where we are we using this file?  Should we document it in `workshop/data/README.md`?

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Thanks @PascalLike.  See minor comments.  Great work!

File: workshop/docker/data/records/example_catalog.geojson (hunk 2 above)
Comment (PascalLike) [reply workshop/docker/data/records/example_catalog.geojson:1]: Sorry, I've added this by mistake... This is for the OGC API Records. I was thinking to create a single geojson with all the records, to have everything in a single collection. But this is still a work in progress, removing this for now. Thanks

**Comment by PascalLike** [comment]
Thanks for the review @tomkralidis . Now it should be fine

**Pull request merged**